volumes/csi-wrapper/hack/azure/README.md

```diff
@@ -0,0 +1,149 @@
+# Azure File CSI Wrapper for Peer Pod Storage
+
+## High Level Design
+
+![design](../../images/csi-wrapper.png)
```

Comment:
do we want to add azure to the picture (source is [here](https://github.com/confidential-containers/cloud-api-adaptor/blob/main/volumes/csi-wrapper/images/csi-wrapper.excalidraw))?

Reply:
Sure, will do.

Reply:
Done